### ISSUE TITLE
Empty lines between @media and child rules 

### ISSUE THREAD

**Issue by juwain** [hudochenkov/postcss-sorting#19]
Thanks for fixing https://github.com/hudochenkov/postcss-sorting/issues/16! There is one more issue I think.

OK, there is an empty line between media-rules now, but there is also an additional line between media-rule and between child rule. I think there should be 1 empty line too.

```
.color {
  @mixin coloring;

  color: white;

  @media (--viewport-md) {
    color: green;
  }

  @media (--viewport-lg) {
    color: red;
  }


  span {
    color: black;
  }

  span {
    color: pink;
  }
}
```

Config is such:
```
{
  "empty-lines-between-children-rules": 1,
  "empty-lines-between-media-rules": 1,
  "sort-order": [
    [
      "@mixin"
    ],
    [
      …
    ],
    [
      "@media"
    ],
    [
      ">child"
    ]
  ]
}

```

**Comment by hudochenkov** [comment]
Sorry about that. Fixed in 1.3.1

**Comment by juwain** [comment]
Thanks! I will PR for atom plugin.
